miso-web/pom.xml

```diff
@@ -252,95 +252,95 @@
-                <jsSourceFile>lims.js</jsSourceFile>
-                <jsSourceFile>search.js</jsSourceFile>
-                <jsSourceFile>experiment_ajax.js</jsSourceFile>
-                <jsSourceFile>library_ajax.js</jsSourceFile>
-                <jsSourceFile>pool_ajax.js</jsSourceFile>
-                <jsSourceFile>project_ajax.js</jsSourceFile>
-                <jsSourceFile>reporting_ajax.js</jsSourceFile>
-                <jsSourceFile>run_ajax.js</jsSourceFile>
-                <jsSourceFile>sample_ajax.js</jsSourceFile>
-                <jsSourceFile>sequencer_partition_container_ajax.js</jsSourceFile>
-                <jsSourceFile>sequencer_reference_ajax.js</jsSourceFile>
-                <jsSourceFile>study_ajax.js</jsSourceFile>
-                <jsSourceFile>parsley_form_validations.js</jsSourceFile>
-                <jsSourceFile>multi_select_drag_drop.js</jsSourceFile>
-                <jsSourceFile>menus.js</jsSourceFile>
-                <jsSourceFile>shared_hot.js</jsSourceFile>
-                <jsSourceFile>jquery/datatables/js/jquery.dataTables.min.js</jsSourceFile>
-                <jsSourceFile>datatables_utils.js</jsSourceFile>
-              </jsSourceFiles>
-              <jsSourceIncludes>
-                <jsSourceInclude>*.js</jsSourceInclude>
-              </jsSourceIncludes>
-              <jsSourceExcludes>
-                <jsSourceExclude>natural_sort.js</jsSourceExclude>
-                <jsSourceExclude>xregexp-all-min.js</jsSourceExclude>
-                <jsSourceExclude>runCalendar.js</jsSourceExclude>
-              </jsSourceExcludes>
-              <jsFinalFile>header_script.js</jsFinalFile>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-compiler-plugin</artifactId>
-      </plugin>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-dependency-plugin</artifactId>
-        <executions>
-          <execution>
-            <id>unpack-fluxion-ajax-js</id>
-            <phase>process-resources</phase>
-            <goals>
-              <goal>unpack</goal>
-            </goals>
-            <configuration>
-              <artifactItems>
-                <artifactItem>
-                  <groupId>uk.ac.tgac.fluxion.ajax</groupId>
-                  <artifactId>fluxion-ajax-js</artifactId>
-                  <version>1.0</version>
-                  <type>jar</type>
-                  <overWrite>true</overWrite>
-                </artifactItem>
-              </artifactItems>
-              <excludes>META-INF</excludes>
-              <outputDirectory>target/ROOT/scripts/fluxion-ajax/</outputDirectory>
-              <overWriteReleases>true</overWriteReleases>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
-    </plugins>
+      <plugins>
```

Comment:
Did the formatter do this? Changes in this file are indentation-only, and the new indentation is incorrect